nixops/script_defs.py

```diff
@@ -893,6 +893,14 @@ def op_show_option(args):
         )
 
 
+def op_eval(args):
+    with deployment(args) as depl:
+        depl.evaluate()
```

Comment:
Not necessarily to be fixed here, but it would be nice to think of a way for the deployment API to not require being careful about calling `evaluate`.